File: Cpp/1470_Shuffle_the_Array.cpp
class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        vector<int> result(2*n);
        for(int i = 0; i < n; i++){
            result[2*i] = nums[i];
            result[2*i+1] = nums[n+i];
        }
        return result;
        /*
        vector<int> result;
        for(int i = 0; i < n; i++){
            result.push_back(nums[i]);
            result.push_back(nums[n+i]);
        }
        return result;
        */
    }
};

File: Python3/1470_Shuffle_the_Array.py
class Solution:
    def shuffle(self, nums: List[int], n: int) -> List[int]:
        """
        x = nums[:n]
        y = nums[n:]
        ans = []
        for i in range(n):
            ans.append(x[i])
            ans.append(y[i])
        return ans
            
        """
        result = [0] * 2 * n
        i = 0
        for i in range(n):
            result[2*i] = nums[i]
            result[2*i+1] = nums[n+i]
        return result
